crates/sui-graphql-rpc/Cargo.toml

```diff
@@ -38,6 +39,8 @@ uuid.workspace = true
 # TODO: put these behind feature flag to prevent leakage
 # Used for dummy data
 bcs.workspace = true
+simulacrum.workspace = true
+sui-transactional-test-runner.workspace = true
```

Comment:
It was easier to leave it here due to usage in pg_integration; which kept making clippy break on my side. But I'll come back and review this; I agree it can be improved